server-timing/load.php

```diff
@@ -6,6 +6,54 @@
  * @since 1.8.0
  */
 
+define( 'PERFLAB_SERVER_TIMING_SETTING', 'perflab_server_timing_settings' );
+define( 'PERFLAB_SERVER_TIMING_SCREEN', 'perflab-server-timing' );
+
+/**
+ * Registers the Server-Timing setting.
+ *
+ * @since n.e.x.t
+ */
+function perflab_register_server_timing_setting() {
+	register_setting(
+		PERFLAB_SERVER_TIMING_SCREEN,
+		PERFLAB_SERVER_TIMING_SETTING,
+		array(
+			'type'              => 'object',
+			'sanitize_callback' => 'perflab_sanitize_server_timing_setting',
+			'default'           => array(),
+		)
+	);
```

Comment:
Not required, but I like to make settings available in the REST API if I am registering the settings anyway. 
```
'show_in_rest' => true,
```

Reply:
Not opposed, but I think if we do that we also should do it for the other setting that the PL plugin registers.